src/iosource/PktSrc.cc

```diff
@@ -82,18 +82,11 @@ void PktSrc::Opened(const Properties& arg_props)
 		return;
 		}
 
-	if ( props.is_live )
-		{
-		Info(util::fmt("listening on %s\n", props.path.c_str()));
-
-		// We only register the file descriptor if we're in live
-		// mode because libpcap's file descriptor for trace files
-		// isn't a reliable way to know whether we actually have
-		// data to read.
-		if ( props.selectable_fd != -1 )
-			if ( ! iosource_mgr->RegisterFd(props.selectable_fd, this) )
-				reporter->FatalError("Failed to register pktsrc fd with iosource_mgr");
-		}
+	Info(util::fmt("listening on %s\n", props.path.c_str()));
```

Comment:
Would keep the "Listening on" message under `props.is_live`. Wouldn't that change the output of `zeek -r` testing (which currently is silent) to a single line saying "listening on /path/to/pcap"?

Reply:
Correct, I thought about adding a "reading from /path/to/pcap" message depending on `props.is_live`. However, this would require adding the path diff-canonifier to some tests and might break external tests as well. So probably not worth it.